### DISCUSSION TITLE
录音软件的设置要求？

### DISCUSSION THREAD

**Discussion by liweigu** [PaddlePaddle/PaddleSpeech#1073] [Q&A] [answered]
直接用录音软件的结果，用预训练模型识别不准，请问需要注意哪些设置？
比如 sample_rate 要设置为 16000。

**Comment by Jackwaterveg** [comment]
使用sample rate 16k， 使用16bit， 单声道的 wav文件

**Comment by liweigu** [comment]
已经设置了“sample rate 16k， 使用16bit， 单声道”，目前还是不准。看看这些设置还有哪些要改？
![设置1](https://user-images.githubusercontent.com/31612697/144575992-fd27292d-8833-4381-a13b-253716db6236.png)
![设置2](https://user-images.githubusercontent.com/31612697/144576019-5f101b98-addd-486e-9802-294f21333bd3.png)
![设置3](https://user-images.githubusercontent.com/31612697/144576154-2b8b6fba-7b0a-4e4e-bde6-06c65c4d971b.png)
![设置4](https://user-images.githubusercontent.com/31612697/144576176-59e4fa6d-4915-4722-aabd-1f812e47af2b.png)
![设置5](https://user-images.githubusercontent.com/31612697/144576196-525cfe7b-d776-431c-aaa4-c823b509b48b.png)
![导出设置1](https://user-images.githubusercontent.com/31612697/144576565-7e67421f-3fa5-4110-996e-86030256ab0e.png)

**Comment by liweigu** [comment]
使用 Cool Edit Pro 录音也不对。
![设置1](https://user-images.githubusercontent.com/31612697/144582809-714cff96-0839-4bc7-94ce-3038e9f6d291.png)
![设置2](https://user-images.githubusercontent.com/31612697/144582824-ce5d2872-9e75-4112-87aa-80a87be0c082.png)
![设置3](https://user-images.githubusercontent.com/31612697/144582861-989f37b4-0836-4214-bd1b-6057c4ee0221.png)
![设置4](https://user-images.githubusercontent.com/31612697/144582869-94d7cbfd-07ca-425b-8f81-c6f895f4021a.png)

**Reply by D-DanielYang** [reply]
是否方便上传一段录音，我们直接试下？

**Comment by liweigu** [comment]
链接：https://pan.baidu.com/s/12QrRlrrncIlvbCo8f_SKpQ 
提取码：1234
麻烦测试一下这个，看看对不对。

**Reply by zh794390558** [reply]
格式没有问题，不过开头有噪音。用的哪个模型？

可以试试用 `paddlespeech asr --input tmp.wav`

**Reply by Jackwaterveg** [reply]
![image](https://user-images.githubusercontent.com/87408988/145327299-bd1ae2b7-3669-4519-8b25-29b37914035f.png)

`paddlespeech asr  --input tmp.wav`

使用该命令的测试结果如上，WER 0.0%。没有啥问题

**Comment by liweigu** [comment, marked as answer]
使用 Cool Edit Pro 录音，“使用sample rate 16k， 使用16bit， 单声道的 wav文件”，即上面上传的文件，用 paddlehub （https://www.paddlepaddle.org.cn/hubdetail?name=u2_conformer_aishell&en_category=AutomaticSpeechRecognition）进行识别，结果是准确的。
